src/pages/GourmetDetail/GourmetDetailReview.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const GourmetDetailReview = props => {
+  const { review } = props;
+  return (
+    <div>
+      {review &&
+        review.map(info => (
+          <DetailReview key={info.id}>
+            <NickName>{info.name}</NickName>
+            <Grade>{info.rating}점</Grade>
+            <CommentBox>
+              <Comment>맛 {info.taste}</Comment>
+              <Comment>가격 {info.price}</Comment>
+              <Comment>서비스 {info.service}</Comment>
+            </CommentBox>
+            <Comment>{info.text}</Comment>
+            <ImageBox>
+              <ReviewImage src={info.img} alt="rivewImg" />
+            </ImageBox>
+          </DetailReview>
+        ))}
+    </div>
+  );
+};
+
+export default GourmetDetailReview;
+
+const DetailReview = styled.div`
+  margin: 30px;
+  padding: 15px;
+  border: 1px solid #f0f0f0;
+  border-radius: 10px;
+`;
+
+const NickName = styled.p`
+  margin: 10px auto;
+  font-size: 20px;
+  font-weight: bold;
+`;
+
+const Grade = styled.div`
+  margin: 10px auto;
+  font-size: 25px;
+`;
+
+const CommentBox = styled.div`
+  display: flex;
+  width: 300px;
+`;
+
+const Comment = styled.p`
+  margin: 10px 20px 10px 0;
+  font-size: 20px;
+`;
+
+const ImageBox = styled.div`
+  display: flex;
+`;
+
+const ReviewImage = styled.img`
+  width: 1200px;
+  height: 300px;
+`;
```

Comment:
```suggestion
ImageBox = styled.div`
  width: 원하는 크기;
  height: 원하는 크기;
  overflow: hidden; // 혹시모를 넘치는 이미지 방지
  
  img {
    width: 100%;
    height: 100%;
    object-fit: cover;
  }
`;
```
이미지 비율을 맞추면서 크기도 원하는 크기로 할 수 있는 코드에요
레이아웃 수정할 때 참고 하세요~
이 코드는 외우시는게 편할꺼에요!

Reply:
@201steve 수정했습니다!